### PR TITLE
updatepicturenames

### LINKED ISSUE #910: Docs image filenames too long for typical ext4 folder structure
### Steps to reproduce
1. be using an ext4 partition for /home
2. mkdir -p /home/test/devel/ && cd /home/test/devel
3. git clone https://github.com/mossmann/hackrf

### Expected behaviour
A cloned repository with successful checkout

### Actual behaviour
A cloned repository with unsuccessful checkout

### Output
```
> git clone https://github.com/mossmann/hackrf
Cloning into 'hackrl'...
...
error: unable to create file docs/images/68747470733a2f2f7261772e6769746875622e636f6d2f6d6f73736d616e6e2f6861636b72662f6d61737465722f646f632f77696b692f696d616765732f6261736562616e642d66696c7465722f6d6178323833372d316d373562772d61742d326d2e706e67.png: File name too long
error: unable to create file docs/images/68747470733a2f2f7261772e6769746875622e636f6d2f6d6f73736d616e6e2f6861636b72662f6d61737465722f646f632f77696b692f696d616765732f6261736562616e642d66696c7465722f6d6178323833372d316d373562772d61742d386d2e706e67.png: File name too long
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.
```

### Version information
Linux, ext4 partition

### PR DESCRIPTION
**Pull request by straithe**
Closes #910

### CODE REVIEW

File: docs/source/tips_tricks.rst [hunk 1]
```diff
@@ -32,12 +32,12 @@ Sampling Rate and Baseband Filters
 
 Using a sampling rate of less than 8MHz is not recommended. Partly, this is because the MAX5864 (ADC/DAC chip) isn't specified to operate at less than 8MHz, and therefore, no promises are made by Maxim about how it performs. But more importantly, the baseband filter in the MAX2837 has a minimum bandwidth of 1.75MHz. It can't provide enough filtering at 2MHz sampling rate to remove substantial signal energy in adjacent spectrum (more than +/-1MHz from the tuned frequency). The MAX2837 datasheet suggests that at +/-1MHz, the filter provides only 4dB attenuation, and at +/-2MHz (where a signal would alias right into the center of your 2MHz spectrum), it attenuates about 33dB. That's significant. Here's a picture:
 
-.. image:: ../images/68747470733a2f2f7261772e6769746875622e636f6d2f6d6f73736d616e6e2f6861636b72662f6d61737465722f646f632f77696b692f696d616765732f6261736562616e642d66696c7465722f6d6178323833372d316d373562772d61742d326d2e706e67.png
+.. image:: ../images/samplepicture1.png
```
Comment (mossmann) [inline docs/source/tips_tricks.rst:35]: Please rename this (and the file) to: max2837-1m75bw-at-2m.png

File: docs/source/tips_tricks.rst [hunk 2]
```diff
@@ -32,12 +32,12 @@ Sampling Rate and Baseband Filters
...
 	:align: center
 
 At 8MHz sampling rate, and using the minimum 1.75MHz bandwidth filter, this is the response:
 
-.. image:: ../images/68747470733a2f2f7261772e6769746875622e636f6d2f6d6f73736d616e6e2f6861636b72662f6d61737465722f646f632f77696b692f696d616765732f6261736562616e642d66696c7465722f6d6178323833372d316d373562772d61742d386d2e706e67.png
+.. image:: ../images/samplepicture2.png
```
Comment (mossmann) [inline docs/source/tips_tricks.rst:40]: Please rename this (and the file) to: max2837-1m75bw-at-8m.png

File: docs/source/tips_tricks.rst [hunk 3]
```diff
@@ -15,12 +15,12 @@ The USB cable you choose can make a big difference in what you see when using yo
...
 Screenshot before and after changing to a noise reducing cable (`view full size image <http://i.imgur.com/e64LASK.jpg>`__): 
 
-.. image:: ../images/687474703a2f2f692e696d6775722e636f6d2f6536344c41534b2e6a7067.jpeg
+.. image:: ../images/noisereducingcablescreenshot.jpeg
 	:align: center
 
 The `cable <http://www.amazon.com/gp/product/B00A99BR90/ref=oh_aui_detailpage_o02_s00?ie=UTF8&psc=1>`__ used when making the screenshot for the difference in noise is: 
 
-.. image:: ../images/687474703a2f2f6563782e696d616765732d616d617a6f6e2e636f6d2f696d616765732f492f34314943364c543361434c2e6a7067.jpeg
+.. image:: ../images/noisereducingcable.jpeg
```
Comment (mossmann) [inline docs/source/tips_tricks.rst:23]: Please remove this image (and the file) and the amazon link, changing the sentence to "A shielded cable with ferrite core was used in the right-hand image."

**Review by mossmann — CHANGES_REQUESTED** [review]
see specific changes requested

File: docs/source/tips_tricks.rst (hunk 3 above)
Comment (straithe) [reply docs/source/tips_tricks.rst:23]: Done.

File: docs/source/tips_tricks.rst (hunk 2 above)
Comment (straithe) [reply docs/source/tips_tricks.rst:40]: Done.

**Review by mossmann — APPROVED** [review]
lovely! thank you!

**Pull request merged**